docs/content/toc.md

```diff
@@ -86,8 +86,7 @@
 ## Development
   * [Overview](../development/overview.html)
   * [Libraries](../development/libraries.html)  
-  * [Extending Druid](../development/modules.html)
```

Comment:
the extensions doc links to it in the section about how to write your own extension